area/area.go

```diff
@@ -0,0 +1,19 @@
+package main
+
+import (
+	"fmt"
+	"math"
+)
+
+func main() {
+	fmt.Println("Enter three triangle sides:")
+	var a, b, c float64
+	fmt.Scanln(&a, &b, &c)
+	var s = (a + b + c) / 2
```

Comment:
nit: In the future please use a short assignment `s := (a + b + c) / 2` instead of `var s = (a + b + c) / 2`.